spec/contracts/roles/create_contract_spec.rb

```diff
@@ -69,7 +69,7 @@
         end
 
         it 'is invalid' do
-          expect_valid(false, type: %i(inclusion))
+          expect_valid(false, type: %i(inclusion inclusion))
```

Comment:
I see it's because of the new type validation on `Role` + the contract's own validations. Perhaps a comment as to why? It wasn't intuitive right off the bat to me; had to try it out 😁